src/Card/CardHeader.jsx

```diff
@@ -21,10 +40,19 @@ CardHeader.propTypes = {
   title: PropTypes.string,
   actions: PropTypes.instanceOf(Object),
   eventManager: PropTypes.instanceOf(Object),
+  themeProps: PropTypes.shape({
+    compose: PropTypes.string,
+    prefix: PropTypes.string,
+  }),
+  theme: PropTypes.instanceOf(Object),
 };
 
 CardHeader.defaultProps = {
   title: undefined,
   actions: undefined,
   eventManager: undefined,
+  themeProps: {
+    compose: 'merge',
```

Comment:
I think I had asked this before, but I do not remember the answer.
why the default  compose is lowercase 'merge' and the one in the code is 'Merge'?